CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [0.6.0] - 2023-11-16
```

Comment:
Nothing in this CHANGELOG indicates that this is a breaking release? Should this be 0.5.1? Or is something missing from the CHANGELOG?

Reply:
> Nothing in this CHANGELOG indicates that this is a breaking release?

This patch introduces **significant** behavioral changes which cannot reasonably be released as a non-breaking change (imo).

Reply:
Fair enough